src/cordova/apple/xcode/ios/Outline.xcodeproj/project.pbxproj

```diff
@@ -373,9 +373,7 @@
 				LastUpgradeCheck = 1010;
 				TargetAttributes = {
 					1D6058900D05DD3D006BFB54 = {
-						DevelopmentTeam = QT8Z3Q9V3A;
```

Comment:
It's weird that the team disappeared. Should I attempt to manually put that back? Should it be in our repo at all??